plugins/draco/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+#
+# Draco plugin CMake configuration
+#
+if (NOT Draco_FOUND)
+    message(FATAL_ERROR "Can't find Draco support required.")
+endif()
+
+if (WIN32)
+    add_definitions(-DNOMINMAX)
```

Comment:
I added this clobber because TileDB was failing too and it was causing test failure churn. I think something might have changed with MSVC 2019's behavior in regard to this because it hadn't been failing in the past.